src/api/operations/referralPosting.fragments.graphql

```diff
@@ -7,3 +7,45 @@ fragment ReferralPostingFields on ReferralPosting {
   status
   assignedDate
 }
+
+fragment ReferralPostingDetailFields on ReferralPosting {
+  id
+  assignedDate
+  chronic
+  denialNote
+  denialReason
+  needsWheelchairAccessibleUnit
+  postingIdentifier
+  referralDate
+  referralIdentifier
+  referralNotes
+  referralResult
+  referredBy
+  referredFrom
+  resourceCoordinatorNotes
+  score
+  status
+  statusNote
+  statusNoteUpdatedAt
+  statusNoteUpdatedBy
+  statusUpdatedAt
+  statusUpdatedBy
+  unitType {
+    id
+    bedType
```

Comment:
We should be displaying the `description`, not the `bedType`. We don't have a use for the "bed type" field yet, which corresponds to the HUD bed types on the HUD Inventory record.